test/testuninitvar.cpp

```diff
@@ -703,6 +714,27 @@ class TestUninitVar : public TestFixture {
                        "  return (*sink)[0];\n"
                        "}");
         ASSERT_EQUALS("", errout.str());
+
+        // Ticket #9296
+        checkUninitVar("int __round_mask(int, int);\n"
+                       "#define round_down(x, y) ((x) & ~__round_mask(y, y))\n"
```

Comment:
I do not like to have preprocessor macros in the test cases. put the preprocessed code here.